usage_record_summary.go

```diff
@@ -0,0 +1,25 @@
+package stripe
+
+// UsageRecordSummary represents a usage record summary.
+// See https://stripe.com/docs/api#usage_records
+type UsageRecordSummary struct {
+	ID               string  `json:"id"`
+	Live             bool    `json:"livemode"`
+	Invoice          string  `json:"invoice"`
```

Comment:
Also, I think for most structs we went with pure alphabetical order — `ID` and `Livemode` don't get special treatment.

Again, it looks like usage records are a little weird in this respect.